Readme.md

```diff
@@ -114,6 +114,12 @@ To install *pnpm* globally run this command: `npm install pnpm -g`. Once it's co
 * *pnpm* will install the latest package dependencies unless it's overriden from the *package.json* file. E.g., the latest *sass* is installed along with the *gulp-dart-sass* that might [cause issues with Bootstrap 4](https://github.com/twbs/bootstrap/issues/34051) so it's overriden with a lower version.
 
 
+## About Git, Line Breaks and Windows
+
+
+For historical reasons, Windows uses the `\r\n` character combination to denote a line break, while Unix-like operating systems such as Linux and macOS simply use a single `\n` character. Git (created by the same guy who made Linux) treats the Unix-style line endings as the only right option. If you are on Windows your Git client is almost certainly configured to "Checkout Windows-style, commit Unix-style" by default to overcome this cultural difference. We disabled the [`linebreak-style`](https://eslint.org/docs/rules/linebreak-style) rule to avoid these headaches. We could've asked you to add a magic string in your repository's _.gitattributes_ files like the ESLint docs did but that's a right headache. [Configure Git](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_formatting_and_whitespace) to ensure your line endings are consistent.
```

Comment:
I just added a note about .gitattributes [here](https://github.com/Lombiq/Gulp-Extensions/blob/dev/Docs/JsTargets.md#:~:text=The%20Lombiq%20rules%20enforce%20CRLF%20line%20endings%20in%20JS%20files.%20To%20ensure%20that%20during%20Git%20checkout%20the%20files%20have%20such%20line%20endings%2C%20you%20can%20add%20the%20followig%20.gitattributes%20file%3A). Why is it "a right headache"?

Reply:
It's a headache because you have to do it per-repository, as opposed to setting up your git linebreak-style which you only need to do per-system and it's the default in most git clients anyway. It's ongoing busywork that people consuming the library (including ourselves) can forget and get frustrated over during initial repo setup.

It's also a weird vendor lock-in that makes no sense when the vast majority of the Git and Javascript ecosystems use Unix-style line endings.

Reply:
It's far from ideal but it's not really an ongoing busywork, rather a one-time setup that you need to do at the time of adding ESLint to your project. I find such explicit configurations better than relying on all consumers to have the right configuration in their git client (including CI systems - which came up today - and third parties), even if it happens per-repository (similarly how we don't rely on everyone having the same config for editors but use an editorconfig).

Can we instead enforce consistent line endings? Ideally, we wouldn't dictate which line ending style somebody uses when working with source files, however, what gets into the repo should be consistently a single one.

Reply:
You get consistent line endings in the repository either way. The difference is only how line endings are handled on the working tree. Using a fixed eol setting means that you won't get native line endings in every OS after checkout. This is a problem because most editors either break or automatically convert the endings to native. So enforcing a fixed local eol type is not a good practice.

If we want to use .gitattributes, it can also be done like this: `* text=auto` ([see](https://stackoverflow.com/a/61283907/14748624))

Reply:
That's good then, thanks. Please modify the .gitattributes recommendation according to this then.